test/gui/shared/steps/steps.py

```diff
@@ -744,3 +744,56 @@ def step(context, resource, content):
 
     print("file has been overwritten")
     waitForFileToBeSynced(context, resource)
+
+
+def enableVFSSupport(vfsBtnText):
+    mouseClick(
+        waitForObjectItem(names.stack_folderList_QTreeView, "_1"),
+        718,
+        27,
+        Qt.NoModifier,
+        Qt.LeftButton,
+    )
+    activateItem(waitForObjectItem(names.settings_QMenu, vfsBtnText))
+    clickButton(
+        waitForObject(names.stack_Enable_experimental_placeholder_mode_QPushButton)
+    )
+
+
+@When("the user enables virtual file support")
+def step(context):
+    enableVFSSupport("Enable virtual file support (experimental)...")
+
+
+@Then('the "|any|" button should be available')
+def step(context, btnText):
+    mouseClick(
+        waitForObjectItem(names.stack_folderList_QTreeView, "_1"),
+        718,
+        35,
```

Comment:
These co-ordinate are very confusing. I may be there should be a single source for the options button `...` . Or atleast you should mention it in comment here saying that this element is present in position (718,35). 🤔